test/testConstants/NewCalcBreakdownUnitTestConstants.scala

```diff
@@ -929,6 +929,14 @@ object NewCalcBreakdownUnitTestConstants {
         income = 100000,
         taxAmount = 40000.00
       ),
+      TaxBands(
```

Comment:
Good to see the test for this AC "Please adhere to existing sorting method
@for(band <- viewModel.lumpSumsBands.getOrElse(Seq()).filter(.income > 0).sortBy(.rate)) { @taxBandTableRow(band, viewModel.taxRegime.contains("Scotland")) }" in both Pay, pensions and profit and Employment lump sums